cmd/dependencyverifier/dependencyverifier.go

```diff
@@ -268,7 +268,7 @@ func main() {
 	}
 	if !bytes.Equal(expected, actual) {
 		log.Printf("Expected status of\n%s", string(expected))
-		log.Printf("Got status of\n%s", string(actual))
+		log.Fatal("Got status of\n", string(actual))
```

Comment:
I find the problem. We did not break when the status is not met here.

That is why I see the `Got status of` diff with `Expected status` log, but some tests are not failing. And this is why the wrong order unwanted dependencies can be merged.